source/dsl/DangerResults.ts

```diff
@@ -208,6 +208,10 @@ export function sortResults(results: DangerResults): DangerResults {
   }
 }
 
+export function isEmpty(results: DangerResults): boolean {
```

Comment:
Any chance this could mention results? `isEmptyResults`? Then this is good to go!